test/box-tap/several-listening-sockets.test.lua

```diff
@@ -65,6 +65,34 @@ for _, path in ipairs(path_table) do
     test:ok(not fio.path.exists(path), "fio.path.exists")
 end
 
+-- Check connection when uris passed in a new common way as an
+-- array of string URIs and table URIs, with different parameters
+local uri = require('uri')
+local uri_table = {
+    "unix/:./tarantoolA?transport=plain",
+    {"unix/:./tarantoolB", params = {transport = "tls"}},
+    {uri = "unix/:./tarantoolC"},
+    default_params = {transport = "plain"}
```

Comment:
These parameters don't have any effect. Please drop this test.

Reply:
ok, fixed